tests/conftest.py

```diff
@@ -12,3 +15,30 @@ def fx_rng() -> Generator:
 @pytest.fixture()
 def fx_bg() -> PCG64:
     return PCG64(SEED)
+
+
+@pytest.fixture
+def hadamardpattern():
+    circ = graphix.transpiler.Circuit(1)
+    circ.h(0)
+    return circ.transpile().pattern
+
+
+@pytest.fixture
+def nqb(fx_rng: Generator):
+    return fx_rng.integers(2, 5)
+
+
+@pytest.fixture
+def depth():
```

Comment:
Fixed in 28f6aaa (the use of `pytest_configure` is documented here: https://docs.pytest.org/en/latest/deprecations.html#pytest-namespace ).

Reply:
If I'm correct, assigning to `pytest.XXX` is "a stopgap measure".
How about using simple constant (ex. `DEPTH=1` and then use `DEPTH` instead of `pytest.depth`)?

Reply:
Fixed in 55331b5.